### PR TITLE
simpler test glob

### PR DESCRIPTION
**Pull request by turadg**
Closes: #1913

## Description

This set out to use the Ava default file glob (i.e. no custom setting) but ultimately doesn't for two reasons:
- requires all the non-test files to have a preceding underscore (this is both laborious to complete and not a unanimously well received outcome)
- would allow test files to have any file name, including `foo.fixture.js`. We like the `.test.js` consistency

Since I did move a bunch of files to use underscores, I left that change in here, along with the script that does it. I have no qualms about removing them at reviewer request.

This also includes a little CI change that saved me time, [ci: no-bail so all packages run lint](https://github.com/endojs/endo/pull/2456/commits/8618d95a8e22b63c4d754b0c45e3ee3aa5b03907)

### Security Considerations

n/a

### Scaling Considerations

n/a

### Documentation Considerations

Consistent with existing practices. There is a lot of underscore now, but it's not required.

### Testing Considerations

CI

### Compatibility Considerations

Not exported

### Upgrade Considerations

none

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,9 +35,9 @@
     "docs:markdown-for-agoric-documentation-repo": "typedoc --plugin typedoc-plugin-markdown --tsconfig tsconfig.build.json",
     "update": "lernaupdate --dedupe",
     "format": "prettier --write .github packages",
-    "lint": "prettier --check .github packages && lerna run lint",
+    "lint": "prettier --check .github packages && lerna run --no-bail lint",
     "lint-fix": "lerna run --no-bail lint-fix",
-    "test": "lerna run --ignore @endo/skel test",
+    "test": "lerna run --ignore @endo/skel --no-bail test",
```
Comment (kriskowal) [inline package.json:40]: `--no-opportunity-for-parole`

**Review by kriskowal — APPROVED** [review]
Love kicking off the legacy test path pattern. Head’s up to @boneskull with possibly-impacted work in flight.

**Comment by kumavis** [comment]
some weird behavior around `--no-bail`

doc https://github.com/lerna/lerna/tree/main/libs/commands/run#--no-bail
issue https://github.com/lerna/lerna/issues/3827

**Comment by kumavis** [comment]
`yarn workspaces foreach` runs all without bailing https://github.com/endojs/endo/pull/2565

**Pull request merged**